server/src/main/java/org/elasticsearch/index/mapper/DateScriptFieldType.java

```diff
@@ -98,7 +97,7 @@ AbstractScriptFieldType<?> createFieldType(
             String pattern = format.getValue() == null ? DateFieldMapper.DEFAULT_DATE_TIME_FORMATTER.pattern() : format.getValue();
             Locale locale = this.locale.getValue() == null ? Locale.ROOT : this.locale.getValue();
             DateFormatter dateTimeFormatter = DateFormatter.forPattern(pattern, supportedVersion).withLocale(locale);
-            return new DateScriptFieldType(name, factory, dateTimeFormatter, script, meta);
+            return new DateScriptFieldType(name, factory, dateTimeFormatter, script, meta, onErrorContinue());
```

Comment:
I think that I would have added an argument to the existing createFieldType method rather than calling the getter here. That's what we do for all other parameters.